libcontainerd/remote_daemon_linux.go

```diff
@@ -31,6 +31,14 @@ func (r *remote) setDefaults() {
 	if r.OOMScore == 0 {
 		r.OOMScore = -999
 	}
+
+	for key, conf := range r.pluginConfs.Plugins {
+		if conf == nil {
```

Comment:
this is a little weird to handle an unset.

Reply:
I liked this more than having a `WithDisablePlugin`, but could do that as an alternative. This isn't a publicly used interface though and I am trying to refactor these in another PR that is already open.